src/routes/login.tsx

```diff
@@ -131,12 +157,26 @@ export function LoginForm() {
                           <Label htmlFor="accessKeyUser">Access Key</Label>
                         </div>
                       </RadioGroup>
-                      <Input
-                        value={key}
-                        type="text"
-                        placeholder={userOption === 'xPriv' ? 'xPriv' : 'Access Key'}
-                        onChange={onChangeKey}
-                      />
+                      <div className="relative">
+                        <Input
+                          value={key}
+                          ref={inputRef}
+                          type={isPasswordVisible ? 'text' : 'password'}
+                          placeholder={userOption === 'xPriv' ? 'xPriv' : 'Access Key'}
+                          onChange={onChangeKey}
+                        />
+                        {isPasswordVisible ? (
+                          <EyeSlashIcon
```

Comment:
Those icons are appearing over the provided xpriv, what makes it unreadable
<img width="374" alt="image" src="https://github.com/user-attachments/assets/ccfeb873-c2a8-471c-822d-249ce2ab183a">

Reply:
Changed